salt/addons/dex/manifests/15-configmap.yaml

```diff
@@ -43,6 +43,36 @@ data:
           groupAttr: uniqueMember
 
           nameAttr: cn
+    {% for con in pillar['dex_connectors'] %}
+    - type: ldap
+      id: "{{ con['id'] }}"
+      name: "{{ con['name'] }}"
+      config:
+        host: "{{ con['server'] }}"
```

Comment:
For most of these variables, replacing this:

    host: "{{ con['server'] }}"

with this:

    host: {{ con['server'] | yaml_dquote }}

would be safer - e.g.if any of the input values contains a ", it would be handled correctly.

Reply:
Agreed that adding dquote filter to the values is a good change. Will do so and update this PR.